src/NexusMods.DataModel/Games/IGame.cs

```diff
@@ -53,10 +53,10 @@ public interface IGame
     /// Stream factory for the game's image, should be close to 16:9 aspect ratio.
     /// </summary>
     public IStreamFactory GameImage { get; }
-    
-    
+
+
     /// <summary>
     /// Collection of <see cref="IGameCapability"/> that this game supports.
     /// </summary>
-    public Dictionary<GameCapabilityId, IGameCapability> SupportedCapabilities { get; }
+    public Dictionary<GameCapabilityId, IGameCapability> SupportedCapabilities();
```

Comment:
Great point, we don't want to populate the dictionary each time, but I wanted to have a cascade structure, where each element in the inheritance could add capabilities. So I guess I'll have to compose the dictionaries instead of just adding the the existing dictionary.

Alternatively, what do you think of using a list instead of a dictionary. We still have the GameCapabilityId inside each capability. We would manly loose out on lookup speed, but would simplify the overall thing.

Reply:
I didn't go with the immutable dictionary in the end since it wasn't really worth it considering I planned to update it in each inheritance child. 
Let me know what you think of the new version. 
If we confirm capabilities as the design of choice going forward I think I'll add a CapabilitiesCollection class with Register() and TryGetCapability() methods.